src/openforms/submissions/models.py

```diff
@@ -436,6 +438,12 @@ def get_merged_data(self) -> dict:
 
         return merged_data
 
+    def _get_value_label(self, possible_values: list, value: str) -> str:
```

Comment:
`self` is unused, so this can be a `@staticmethod`

I think in the longer term we should look into a system to define data formatters/getters, something like:

```py
@formatter("select")
def get_select_repr(component: dict, value: Any) -> str:
    values = component["data"]["values"]
    ...
    return foo
```

& then have fuctions for the `formio` component types.